### ISSUE TITLE
Does redundant entry require saving PII?

### ISSUE THREAD

**Issue by priyankad7** [w3c/wcag#1843]
There may be a potential for user PII to have to be retained in scenarios where a user is required to step through multiple pages to complete a particular form or task (since certain information will need to be retained in session as the user navigates from page to page in this rule).  This may result in either a security risk to the user or a requirement to change data classification on web pages that otherwise would not be subject to PII / privacy requirements.  

I'm unsure how we would solve either issue – even if simply using a session-state cookie to retain values (or if it is, in fact, a valid concern).  This is more about raising a potential issue so it can be considered.

**Comment by mraccess77** [comment]
I'd think this could fall into the exception (but I think we could clarify this more in the understanding document perhaps):

"the information is required to ensure the security of the content, or"

**Comment by alastc** [comment]
Proposed response: 

>  This may result in either a security risk to the user or a requirement to change data classification on web pages that otherwise would not be subject to PII / privacy requirements.

We are not creating a requirement to retain that information across all pages, it just has to be available when requested.

> even if simply using a session-state cookie to retain values (or if it is, in fact, a valid concern).

Keeping values in the cookie would not be a secure way of retaining data, that should be server-side. Generally any application that retains information will just use a cookie to identify the user, and loading the page will request the relevant data.

In an SPA the data may be retained in local-storage (or equivalent), regardless of whether it was requested on other pages.
